workbench-core/accounts/src/services/accountService.ts

```diff
@@ -33,21 +39,30 @@ export default class AccountService {
   }
 
   /**
-   * Get all account entries from DDB
+   * Get account entries from DDB
    *
    * @returns Account entries in DDB
    */
-  public async getAccounts(): Promise<Account[]> {
-    const queryParams = {
-      index: 'getResourceByCreatedAt',
-      key: { name: 'resourceType', value: 'account' }
+  public async getAccounts(queryParams: QueryParams): Promise<PaginatedResponse<Account>> {
+    const response = await this._dynamoDBService.getPaginatedItems(queryParams);
+
+    return {
+      data: response.data.map((item) => {
+        return AccountParser.parse(item);
+      }),
+      paginationToken: response.paginationToken
     };
+  }
 
-    const response = await this._aws.helpers.ddb.getPaginatedItems(queryParams);
-
-    return response.data.map((item) => {
-      return AccountParser.parse(item);
-    });
+  public async getAccountsForAccountHandler(queryParams: QueryParams): Promise<Account[]> {
```

Comment:
Thoughts on name this `getAllAccounts`. We should name the function by what it does instead of by who is currently using it. This function doesn't have to be exclusively used by `AccountHandler`.